payments/admin.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib import admin
+from django.utils.translation import ugettext_lazy as _
+
+from payments.models import (
+    AdditionalProduct,
+    BerthPriceGroup,
+    BerthProduct,
+    WinterStorageProduct,
+)
+
+
+class BerthPriceGroupAdmin(admin.ModelAdmin):
+    readonly_fields = ("berth_types",)
+
+    def berth_types(self, obj):
```

Comment:
can this be `filter_horizontal` instead of the readonly field?

Reply:
Don't think it will be possible
```
<class 'payments.admin.BerthPriceGroupAdmin'>: (admin.E020) The value of 'filter_horizontal[0]' must be a many-to-many field.
```

Reply:
yeah... maybe we should try some inline here? I would like to have some kind of selection interface here to avoid setting it from the berth type page.

Reply:
I mean, you should also be able to set it on the berth type page, but it is easier to just choose several berth types from this one page

Reply:
Yeah, I thought about inlining as well. The problem is that it only lets you create new BerthTypes, not select existing ones, but I can dig a bit deeper to try and find something.

Reply:
yes, let's keep it as read-only for now, since we are assigning / generating BPGs automatically for now

Reply:
🆒 🆒 🆒